src/main/scala/com/gu/identity/play/CookieBuilder.scala

```diff
@@ -0,0 +1,33 @@
+package com.gu.identity.play
+import org.joda.time.{DateTime, Duration}
+import play.api.libs.functional.syntax._
+import play.api.libs.json.Reads._
+import play.api.libs.json._
+import play.api.mvc.Cookie
+
+import scala.util.Try
+
+object CookieBuilder {
+  def fromGuestConversion(json: JsValue, domain: Option[String] = None): Option[Seq[Cookie]] = {
+    implicit val cookieRead: Reads[Cookie] = (
+      (JsPath \ "key").read[String] and
+        (JsPath \ "value").read[String]
+      )(Cookie.apply(_, _))
+
+    for {
+      expirationString <- (json \ "cookies" \ "expiresAt").asOpt[String]
+      expiration <- Try { new DateTime(expirationString) }.toOption
+      maxAge = new Duration(DateTime.now, expiration).getStandardSeconds.toInt
+      cookies <- (json \ "cookies" \ "values").asOpt[Seq[Cookie]]
+      guuCookie <- cookies.find(_.name == "GU_U")
+      scguuCookie <- cookies.find(_.name == "SC_GU_U")
+      scgulaCookie <- cookies.find(_.name == "SC_GU_LA")
```

Comment:
> Yes, we have this SC_ rule, but how do we decide which one has a max-age for instance ?

That particular bit of information is already returned by the Identity API with a `sessionCookie` flag:

https://github.com/guardian/identity/blob/bb4a7611e/identity-api/src/main/scala/com/gu/identity/api/services/CookieResponseFactory.scala#L37

https://github.com/guardian/frontend/blob/fd2614c1/identity/app/idapiclient/responses/CookiesResponse.scala#L5-L7

...so you can use that!